src/Workspaces/Core/Portable/Workspace/Host/SolutionServices.cs

```diff
@@ -8,8 +8,10 @@
 
 namespace Microsoft.CodeAnalysis.Host
 {
-    // TODO(cyrusn): Make public.  Tracked through https://github.com/dotnet/roslyn/issues/62914
-    internal sealed class HostSolutionServices
+    /// <summary>
+    /// Per solution services provided by the host environment.
+    /// </summary>
+    public sealed class SolutionServices
```

Comment:
new public API.  it's worth expanding this file to see the entire shape exposed.